docker-compose.yaml

```diff
@@ -1,13 +1,14 @@
 services:
-  ? ffc-sfd-file-receiver
-  : build:
+   ffc-sfd-file-receiver: 
```

Comment:
Set up looks good. My only question is on the naming of the service/repository as while. Our other receiver services are named:

- `ffc-sfd-customer-receiver-messages`
- `ffc-sfd-customer-receiver-payments`
- `ffc-sfd-customer-receiver-preferences`
- `ffc-sfd-customer-receiver-queries`

So I would imagine this service should be named `ffc-sfd-customer-receiver-files` instead of `ffc-sfd-file-receiver`?

Reply:
Our other service is named : sfd-file-processor, because this is a component(essentially data orchestration helper) of file-processing, we have taken a decision for this naming: file-xxx, happy to discuss if renaming will provide any benefits, but keep in mind that we will then need to rename file-processor to have customer prefix too. Not sure how easy this is done, taking into account that we will need to change ADO stuff too?

Reply:
You wouldn't need to rename the file processor to include the `customer` prefix as the naming convention only applies to the receivers. For example we have a service called `ffc-sfd-messages-processor` which is similar to `ffc-sfd-file-processor`, but the corresponding messages receiver is named `ffc-sfd-customer-receiver-messages`? However since you said the naming convention was already agreed I'm happy to approve.

Reply:
Correction: have already approved this PR since you already created the repo and have done the setup so go ahead and merge once your happy 👍🏽